support-frontend/assets/helpers/tracking/quantumMetric.ts

```diff
@@ -128,6 +138,68 @@ function sendEventWhenReadyTrigger(sendEventWhenReady: () => void): void {
 	}
 }
 
+function sendEventAcquisitionDataFromQueryParamEvent(
+	acquisitionData: ReferrerAcquisitionData,
+): void {
+	void ifQmPermitted(() => {
+		const sendEventWhenReady = () => {
+			const source = acquisitionData.source;
+			const componentId = acquisitionData.componentId;
+			const componentType = acquisitionData.componentType;
+			const campaignCode = acquisitionData.campaignCode;
+			const referrerUrl = acquisitionData.referrerUrl;
+			const isRemote = acquisitionData.isRemote;
+
+			if (window.QuantumMetricAPI?.isOn()) {
+				if (source) {
+					sendEvent(
+						SendEventAcquisitionDataFromQueryParam.Source,
+						false,
+						source.toString(),
+					);
+				}
+				if (componentId) {
+					sendEvent(
+						SendEventAcquisitionDataFromQueryParam.ComponentId,
+						false,
+						componentId.toString(),
+					);
+				}
+				if (componentType) {
+					sendEvent(
+						SendEventAcquisitionDataFromQueryParam.ComponentType,
+						false,
+						componentType.toString(),
+					);
+				}
+				if (campaignCode) {
+					sendEvent(
+						SendEventAcquisitionDataFromQueryParam.CampaignCode,
+						false,
+						campaignCode.toString(),
+					);
+				}
+				if (referrerUrl) {
+					sendEvent(
+						SendEventAcquisitionDataFromQueryParam.ReferrerUrl,
+						false,
+						referrerUrl.toString(),
+					);
+				}
+				if (isRemote) {
```

Comment:
We'd never call `sendEvent` when `isRemote` is `false` by using the conditional here. The suggested code [above](https://github.com/guardian/support-frontend/pull/5434/files#r1393028862) avoids this by converting `toString()` before running the equivilanr conditional